swatch-contracts/src/main/java/com/redhat/swatch/contract/model/ContractEntityMapper.java

```diff
@@ -128,7 +127,7 @@ default String extractBillingProviderId(PartnerEntitlementV1 entitlement) {
       // for contracts created before ITPART-1180, the client ID is not provided,
       // so we need to default it to empty.
       var azureClientId =
-          Optional.ofNullable(entitlement.getPartnerIdentities().getClientId()).orElse(EMPTY);
+          Optional.ofNullable(entitlement.getPartnerIdentities().getClientId()).orElse("");
```

Comment:
Is this a leftover?

Reply:
It was actually a constant pulled in from an external library.